src/main/java/pl/zaprogramujzycie/mma/controllers/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,86 @@
+package pl.zaprogramujzycie.mma.controllers;
+
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.data.crossstore.ChangeSetPersister;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+import pl.zaprogramujzycie.mma.utils.exceptions.ErrorMessage;
+
+import java.util.Date;
+
+
+@ControllerAdvice
```

Comment:
@RestControllerAdvice
I propnuje to dac w pakiecie exceptions, to bardziej obslug wyjatkow niz kontroler (mylaca nazwa 🗡️ )

Reply:
final brakuje